### PR TITLE
Sierra/partners polish

### PR DESCRIPTION
**Pull request by sierrakn**
## What

Polish the partners page and make it mobile responsive

## Show Me

image
image
image
image
image

### CODE REVIEW

File: pages/volunteers.tsx [hunk 1]
```diff
@@ -85,7 +85,7 @@ const VolunteerPage = () => {
             alignItems: 'center',
             gap: { xs: 4, md: '40px' },
           }}
-          maxWidth={'880px'}
+          maxWidth={'1000px'}
```
Comment (santina) [inline pages/volunteers.tsx:88]: per Kylie's suggestion it should be 880 https://www.figma.com/file/Qa1UC7cGLmEXbWXuRmgjpO/OS-Website-Annotations?type=whiteboard&node-id=136-244&t=R7Ic16ISjJQafpE5-0

setting it to 1000px would make the hero section not align with the content of the page.

**Review by santina — CHANGES_REQUESTED** [review]
hey Sierra, looks good to me but just make sure to address Kylie's two comments https://www.figma.com/file/Qa1UC7cGLmEXbWXuRmgjpO/OS-Website-Annotations?type=whiteboard&node-id=136-244&t=R7Ic16ISjJQafpE5-0

File: pages/volunteers.tsx (hunk 1 above)
Comment (mistyb01) [reply pages/volunteers.tsx:88]: I would suggest getting the max width right from the theme file! As in `theme.breakpoints.values.lg`. Doing that on all pages' maxWidths would help w/ consistency

**Comment by gussmith23** [comment]
@sierrakn Did you address Santina's comments on this PR?

**Review by mistyb01 — APPROVED** [review]
I addressed Kylie's feedback with my commits, and the rest looks good. I'll go ahead and merge it.

**Pull request merged**